packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -446,6 +472,10 @@ export class RunningSummarizer implements IDisposable {
                     });
                     await delay(delaySeconds * 1000);
                 }
+
+                if (this.refreshSummaryAckLock !== undefined) {
```

Comment:
You can remove IF and make it unconditional - it's totally Ok to await undefined. :)

Reply:
Even better!